.github/workflows/run-tests.yml

```diff
@@ -26,7 +26,6 @@ jobs:
       - uses: pnpm/action-setup@v4
         name: Install pnpm
         with:
-          version: 8
```

Comment:
This is the build fix, used to compete with the version pinned in our package.json. This script knows how to read the package.json if unspecified